tests/test_mrml.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+
+from django.template import Context, Template
+from django.utils.safestring import mark_safe
+
+from wagtail_newsletter.templatetags.wagtail_newsletter import MRMLError
+
+
+def test_render_mjml():
+    template = Template(
+        """
+        {% load wagtail_newsletter %}
+        {% mrml %}
+            <mjml>
+                <mj-body>
+                    {{ message }}
+                </mj-body>
+            </mjml>
+        {% endmrml %}
+        """
+    )
+    message = "<em>Hello MJML!</em>"
+    html = template.render(Context({"message": mark_safe(message)}))  # noqa: S308
+    assert message in html
+    # Since this was rendered for email, it should have some weird markup inside.
+    assert "<o:OfficeDocumentSettings>" in html
```

Comment:
I wonder if snapshot testing couldn't make sense for this?
(powered by Syrupy, for example)

But equally, we don't want to test MRML itself so maybe it's actually a bad idea :sweat_smile:

Reply:
Yeah, I think we're just testing that this function call works: https://github.com/wagtail/wagtail-newsletter/blob/5b4adc5de380ddfc0f1aaf2610475b93af832ec2/wagtail_newsletter/templatetags/wagtail_newsletter.py#L35

The only reason I've added the `<o:OfficeDocumentSettings>` check is to make sure there's some MJML processing going on, vs. just having the MJML source code returned back unchanged.